packages/wallets/component-tests/wallets-carousel-content.spec.tsx

```diff
@@ -56,7 +56,7 @@ test.describe('Wallets - Mobile carousel', () => {
         });
     });
 
-    test('renders cards for all wallets', async ({ baseURL }) => {
+    test.skip('renders cards for all wallets', async ({ baseURL }) => {
```

Comment:
why skip this test?

Reply:
its failing because the timer is being exceeded, and I asked @wojciech-deriv and he says its alright